src/sentry/eventstream/kafka/backend.py

```diff
@@ -137,18 +144,39 @@ def _send(
     def requires_post_process_forwarder(self):
         return True
 
+    def reset_batch(self) -> None:
+        self._pending_futures.clear()
+        self._batch_create_time_ms = int(time.time() * 1000)
+
+    @staticmethod
+    def compute_metrics(partition: int, task_kwargs: Mapping[str, Any]) -> None:
+        if task_kwargs["group_id"] is None:
+            metrics.incr(
+                "eventstream.messages",
+                tags={"partition": partition, "type": "transactions"},
+            )
+        else:
+            metrics.incr(
+                "eventstream.messages",
+                tags={"partition": partition, "type": "errors"},
+            )
+
     def run_post_process_forwarder(
         self,
         consumer_group,
         commit_log_topic,
         synchronize_commit_group,
         commit_batch_size=100,
+        commit_batch_timeout_ms=1000,
         initial_offset_reset="latest",
     ):
         logger.debug("Starting post-process forwarder...")
-
         cluster_name = settings.KAFKA_TOPICS[settings.KAFKA_EVENTS]["cluster"]
 
+        current_concurrency = options.get("post-process-forwarder:concurrency")
```

Comment:
I'm not sure how our options are typed, might be worth asserting it's an int?

Reply:
Options get typed during registration time. I tried setting it to float and it barfed.